### ISSUE TITLE
HttpSender has only private constructors

### ISSUE THREAD

**Issue by robert4os** [google/identity-toolkit-java-client#35]
Has only private constructors, which are unaccessible outside the library.

**Comment by Erni** [comment]
I think this is not an issue. Why would you want to construct an HttpSender object? It is meant to be used by the GitkitClient Builder. Not out of here, that's why the constructors are protected.
In any case @liujin-google can give you a better answer in case I am wrong.

**Comment by robert4os** [comment]
You are right! I have now forked the project and extended GitkitClient to support email verification status. Thank you.
